source/speech/__init__.py

```diff
@@ -1071,10 +1186,17 @@ def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=cont
 			speak(speechSequence,priority=priority)
 			return True
 
-def getSpeechTextForProperties(reason=controlTypes.REASON_QUERY,**propertyValues):
+
+# C901 'getPropertiesSpeech' is too complex
+# Note: when working on getPropertiesSpeech, look for opportunities to simplify
+# and move logic out into smaller helper functions.
+def getPropertiesSpeech(  # noqa: C901
+		reason=controlTypes.REASON_QUERY,
+		**propertyValues
+) -> SpeechSequence:
 	global oldTreeLevel, oldTableID, oldRowNumber, oldRowSpan, oldColumnNumber, oldColumnSpan
-	textList=[]
-	name=propertyValues.get('name')
+	textList: List[str] = []

```

Comment:
This is now a list of strings, but strictly spoken, it may contain commands in the future. If we introduce commands here, we would need to rename textList to sequence or something. Probably best to do this now?

Reply:
This is true, but it's introducing unnecessary changes and hopefully having to change the name when introducing commands will encourage more thought into the change in type that is occuring.